potodo/arguments_handling.py

```diff
@@ -77,9 +77,23 @@ def check_args(
         # Disable all logging
         logging.disable(logging.CRITICAL)
 
+    excluded_paths = []
+    for p in [Path(path).resolve() for path in exclude]:
+        if p.exists():
+            excluded_paths.append(p)
+        else:
+            print("Potodo: Path `{}` doesn't exist.".format(str(p)))
+            exit(1)
+
+    if path:
+        for p in excluded_paths:
+            if not str(p).startswith(str(path)):
```

Comment:
I'd like to see 
```suggestion
            if not p.is_relative_to(path):
```
but it's Python 3.9 only :(

Reply:
Yeah and potodo is `>= 3.6`